src/Umbraco.Cms.Api.Management/Controllers/AuditLog/ByTypeAuditLogController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Common.ViewModels.Pagination;
+using Umbraco.Cms.Api.Management.Factories;
+using Umbraco.Cms.Api.Management.ViewModels.AuditLogs;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Api.Management.Controllers.AuditLog;
+
+public class ByTypeAuditLogController : AuditLogControllerBase
+{
+    private readonly IAuditService _auditService;
+    private readonly IAuditLogViewModelFactory _auditLogViewModelFactory;
+
+    public ByTypeAuditLogController(IAuditService auditService, IAuditLogViewModelFactory auditLogViewModelFactory)
+    {
+        _auditService = auditService;
+        _auditLogViewModelFactory = auditLogViewModelFactory;
+    }
+
+    [HttpGet("ByType")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(PagedViewModel<AuditlogViewModel>), StatusCodes.Status200OK)]
+    public async Task<IActionResult> ByType(AuditType logType, DateTime? sinceDate = null, int skip = 0, int take = 100)
```

Comment:
There seems to be an issue with this one, when I request it the date is always min value.

Additionally, I think it would be a good idea to maybe add the key to the ViewModel, since it'll make this endpoint a lot more useful.